sdk_container/src/third_party/coreos-overlay/app-emulation/containerd/files/containerd.service

```
@@ -3,7 +3,7 @@ Description=Containerd Container Daemon
 Documentation=http://github.com/docker/containerd
 
 [Service]
-Type=simple
+Type=notify
```

I think we should take this opportunity to align it more with upstream:
https://github.com/containerd/containerd/blob/main/containerd.service
```suggestion
Type=notify
KillMode=process
RestartSec=5
OOMScoreAdjust=-999
LimitNOFILE=infinity
ExecStartPre=-/sbin/modprobe overlay
```
(different order and without the LimitNOFILE duplicate below)